login-layout/ligin.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="login.css">
+    <title>login page</title>
+</head>
+
+<body>
+    <div class="main-container">
+        <header>
+            <nav class="navigation-bar">
+                <a class="title" href="/index.html">neon camera</a>
+                <input class="search-bar" type="text" placeholder="&#128269; Search">
+                <ul class="side-nav-section">
+                    <li><button class="login-btn"><a href="/login-layout/ligin.html">Login</a></button></li>
+                    <li><a class="wishlist-btn btn" href="/wish-list/wish-list.html">&#9825;<span
+                                class="badge-wishlist">2</span></a></li>
+                    <li><a class="add-cart-btn btn" href="/my-cart/my-cart.html"> &#x1f6d2; <span
+                                class="badge-wishlist">2</span></a></li>
+                </ul>
+            </nav>
+        </header>
+        <form class="login-main-container">
+            <div class="login-container">
+                <h1 class="login-heading">Login</h1>
+                <section class="mail-password-container">
+                    <label for="/">Email address</label>
+                    <input class="mail-input" type="email" placeholder="abc@gmail.com">
+                    <label for="/">Password</label>
+                    <input class="pass-input" type="password" placeholder="*************">
+                </section>
+                <section class="forget-password-section">
+                    <label for="/">
+                        <input class="checkbox-inpt" type="checkbox">
+                        Remember me</label>
+                    <a class="forget-pass-link" href="/">Forget Password?</a>
+                </section>
+                <button class="login-btn">Login</button><br>
+                <a class="create-account-link" href="/">Create New Account ></a>
+                <div class="signup-way-container">
+                    <a href="https://www.facebook.com/" target="_blank"><img class="logo-img"
+                            src="/login-layout/images/fb.png" alt=""></a>
```

Comment:
Suggestion

- You can give alt to all your images  to improve the accessibility features

Reply:
ok, I have added.